test/unit/specs/snapshotter.spec.js

```diff
@@ -0,0 +1,48 @@
+var Map = require('ti.map');
+const ANDROID = Ti.Platform.osname === 'android';
+if (!ANDROID) {
+	var snapshotter = Map.createSnapshotter({
+		mapType: Map.NORMAL_TYPE,
+		region: {
+			latitude: 49.261902,
+			longitude: -123.12494706,
+			latitudeDelta: 0.4,
+			longitudeDelta: 0.4,
+		},
+		size: { height: 200, width: 300 },
+		showsBuildings: true,
+		showsPointsOfInterest: false,
+	});
+
+	describe('ti.map.Snapshotter', function () {
```

Comment:
@anil-shukla-axway, its better to add a describe tag which can tell wether UT is for methods, property or constraint. Something like

`describe('methods', () => {
}`

Reply:
Done